src/platform/user/authorization/containers/MHVApp.jsx

```diff
@@ -7,6 +7,7 @@ import appendQuery from 'append-query';
 import AlertBox from '@department-of-veterans-affairs/formation/AlertBox';
 import LoadingIndicator from '@department-of-veterans-affairs/formation/LoadingIndicator';
 import { mhvAccessError } from '../../../static-data/error-messages';
+import backendServices from '../../../../platform/user/profile/constants/backendServices';
```

Comment:
Might be going up a couple extra levels for this path; this is already in `platform/user`.

Reply:
Ah, good call! Thanks!